openapi-cli/src/main/java/io/ballerina/openapi/generators/client/BallerinaAuthConfigGenerator.java

```diff
@@ -646,6 +694,29 @@ public AssignmentStatementNode getApiKeyAssignmentNode() {
         return recordFieldNodes;
     }
 
+    /**
+     * Generate fields for`AuthConfig` record when combination of ApiKey and HTTPOrOAuth authentication is supported.
+     *
+     * <pre>
+     *     # Auth Configuration
+     *     http:BearerTokenConfig|http:OAuth2RefreshTokenGrantConfig|ApiKeysConfig auth;
+     * </pre>
```

Comment:
Shall we add extra line after the function description ?

Reply:
Done